From b05ab0eaebd9e6e58f1b9c092e05323c5b901a2b Mon Sep 17 00:00:00 2001
From: Alexander Tsoy <alexander@tsoy.me>
Date: Mon, 27 Oct 2014 16:45:10 +0300
Subject: [PATCH] build: Workaround quoting issues in configure script

Default value of BROWSER_PLUGIN_DIR variable contains special symbols.
Thus quoting and inlining it in parameter expansion is not portable.
In particular it does not work in dash. Replace ${a:-b} parameter
expansion with conditional statement.

https://bugzilla.gnome.org/show_bug.cgi?id=739241
---
 configure.ac | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index d55848b..5b8c306 100644
--- a/configure.ac
+++ b/configure.ac
@@ -232,7 +232,9 @@ esac
 AM_CFLAGS="$AM_CFLAGS $WARN_CFLAGS"
 AC_SUBST(AM_CFLAGS)
 
-BROWSER_PLUGIN_DIR="${BROWSER_PLUGIN_DIR:-"\${libdir}/mozilla/plugins"}"
+if test -z "${BROWSER_PLUGIN_DIR}"; then
+  BROWSER_PLUGIN_DIR="\${libdir}/mozilla/plugins"
+fi
 AC_ARG_VAR([BROWSER_PLUGIN_DIR],[Where to install the plugin to])
 
 AC_CONFIG_FILES([
-- 
2.1.3

